frontend/src/components/staff/CreatePackage.jsx

```diff
@@ -155,7 +158,7 @@ export default function CreatePackage({ refresh, onRefresh }) {
           'Content-Type': 'application/json',
         },
         body: JSON.stringify(postData),
-      });
+      }).catch(() => {});
```

Comment:
please refer to this comment for the empty catches:

instead of an empty catch block it'd be helpful if the user could see what went wrong on their screen instead of no feedback. thus, let's try a notification like the one on line 162 of this file in each catch with a failure message describing what error we caught